docs/sessions/2026-05-21-surface-parity.md

```
@@ -0,0 +1,158 @@
+date: 2026-05-21 03:27:00 EST
+repo: https://github.com/jmagar/syslog-mcp
+branch: feat/surface-parity
+head: 2025995
+plan: docs/superpowers/plans/2026-05-21-surface-parity.md
+agent: Claude (claude-sonnet-4-6)
+working directory: /home/jmagar/workspace/syslog-mcp/.worktrees/surface-parity
+worktree: /home/jmagar/workspace/syslog-mcp/.worktrees/surface-parity  2025995 [feat/surface-parity]
+pr: "#40 — feat: surface parity — CLI + REST API for all MCP actions — https://github.com/jmagar/syslog-mcp/pull/40"
+---
+
+## User Request
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)